regexes/device/mobiles.yml

```
@@ -14588,8 +14819,136 @@ Macoox:
   device: 'smartphone'
   model: 'MC-X7 Mini'
 
-# Red
+# Prixton
+Prixton:
+  regex: '(T7012Q)[);/ ]'
+  device: 'tablet'
+  model: '$1'
+
+# Newland
+Newland:
+  regex: '(NLS-MT90)[);/ ]'
```

We should maybe try to avoid using matches where they are not required

```suggestion
  regex: '(?:NLS-MT90)[);/ ]'
```